src/main/java/games/strategy/engine/framework/ArgParser.java

```diff
@@ -18,7 +21,13 @@ public static boolean handleCommandLineArgs(
     if (args.length == 1 && !args[0].contains("=")) {
       // assume a default single arg, convert the format so we can process as normally.
       if (args[0].startsWith(TRIPLEA_PROTOCOL)) {
-        args[0] = GameRunner.TRIPLEA_MAP_DOWNLOAD_PROPERTY + "=" + args[0].substring(TRIPLEA_PROTOCOL.length());
+        final String encoding = StandardCharsets.UTF_8.displayName();
+        try {
+          args[0] = GameRunner.TRIPLEA_MAP_DOWNLOAD_PROPERTY + "="
+              + URLDecoder.decode(args[0].substring(TRIPLEA_PROTOCOL.length()), encoding);
+        } catch (UnsupportedEncodingException e) {
+          throw new IllegalStateException(encoding + " is not a supported encoding!", e);
```

Comment:
From our earlier discussion of different [types of failure modes](https://github.com/google/guava/wiki/ConditionalFailuresExplained), I would say this falls under the _Impossible Condition_ check.  That is, there is something fundamentally wrong with the JVM if UTF-8 is not supported.  So I would consider throwing `AssertionError` here instead.

Also, a Clean Up will mark the `catch` parameter on the previous line as `final`.